### PR TITLE
Translate `pipeline_tutorial.md` to chinese

### LINKED ISSUE #26803: [i18n-zh] Translating docs to Chinese (Simplified)
Hi!

Let's bring the documentation to all the <languageName>-speaking community 🌐 (currently 0 out of 267 complete)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `<languageCode>` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `<languageCode>/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

And I can see some part of the work has be done (Great work!). I just list all files below.

## Get Started section
- [X] [index.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.md) https://github.com/huggingface/transformers/pull/20180
- [X] [quicktour.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.md) (waiting for initial PR to go through)
- [X] [installation.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.md).

## Tutorial section
- [ ] [pipeline_tutorial.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.md)
- [ ]  [autoclass_tutorial.md](https://github.com/huggingface/transformers/blob/master/docs/source/autoclass_tutorial.md)
- [ ]  [preprocessing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.md)
- [ ]  [training.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.md)
- [X]  [accelerate.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.md)
- [ ]  [model_sharing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.md)
- [ ]  [multilingual.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.md)

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?
Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu

I just updated my translation work. Here is a guileline during my translation:
- Keep some terminology in English format. Those workds appears in amounts of class files or other documents. I think it's better to keep it in original format rather give a chinese translation.

File: docs/source/zh/pipeline_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+</Tip>
+
+[链接](./pipeline_webserver)
+
+## 视觉流水线
+
+对于视觉任务，使用[`pipeline`] 几乎是相同的。
+
+指定您的任务并将图像传递给分类器。图像可以是链接、本地路径或base64编码的图像。例如，下面显示的是哪种品种的猫？
+
```
Comment (stevhliu) [inline docs/source/zh/pipeline_tutorial.md:217]: Missing the image link here:

```suggestion
![pipeline-cat-chonk](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg)
```

**Review by stevhliu — COMMENTED** [review]
Thanks, that makes sense to me! Last step is to add this file to the `toctree` so it gets built :)

File: docs/source/zh/pipeline_tutorial.md (hunk 1 above)
Comment (jiaqiw09) [reply docs/source/zh/pipeline_tutorial.md:217]: fix it. Thanks for your reviw

**Comment by jiaqiw09** [comment]
@stevhliu Thansk for your review. I just fix these two problems. Would you mind having a check again?
And for another PR of preprocessing.md, I will update it after this PR is merged. Both PRs changes `_toctree.yml`, there may be some merge conflict problem.
Best

**Review by stevhliu — APPROVED** [review]
Great work, thanks again!

**Pull request merged**